db/migrate/20240420053540_add_account_id_not_null_constraint_for_licenses.rb

```diff
@@ -0,0 +1,9 @@
+class AddAccountIdNotNullConstraintForLicenses < ActiveRecord::Migration[7.1]
+  def up
+    log_level_was, ActiveRecord::Base.logger.level = ActiveRecord::Base.logger.level, Logger::DEBUG
```

Comment:
Should we extract this out into an extension? Seems like we would want to use it for most migrations.

Reply:
Squatting https://rubygems.org/gems/verbose_migrations for this. Right now, thinking of a simple API like:

```rb
class AddAccountIdNotNullConstraintForLicenses < ActiveRecord::Migration[7.1]
  verbose! # or: enable_verbose_logging!

  def up
    add_check_constraint :licenses, 'account_id IS NOT NULL', name: 'licenses_account_id_not_null', validate: false
  end
end
```